tests/integration/helpers.py

```diff
@@ -858,8 +855,8 @@ async def check_tls_patroni_api(ops_test: OpsTest, unit_name: str, enabled: bool
                 # TLS is enabled, otherwise True is set because it's the default value
                 # for the verify parameter.
```

Comment:
most probably comment should be adjusted

Reply:
Updated to reflect current behaviour in https://github.com/canonical/postgresql-operator/pull/909/commits/0c73c024046826930a714810dce1c60de74c7ad1